### ISSUE TITLE
Plain links in documentation will generate console errors since 6.1.0

### ISSUE THREAD

**Issue by robcaldecott** [styleguidist/react-styleguidist#793]
Reproduced here:

https://github.com/robcaldecott/styleguidist-links

To reproduce just include a plain URL in any markdown or component comments. These will be stripped and the browser will display console errors.

This started happening with v6.1.0.

For example:

```js
/**
 * The only true button.
 *
 * https://github.com/robcaldecott/styleguidist-links
 */
```

Errors look like this (Chrome/Windows 10):

```bash
Warning: Unknown props `github`, `com`, `robcaldecott`, `styleguidist-links` on <https:> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
    in https: (created by Markdown)
```

**Comment by nitinkh** [comment]
Hi @robcaldecott : I think it is not a bug, if we want to get our URL reflected, we need to use markdown's links syntax. So for your case we can use it like : 
/**
 * The only true button.
 * 
 * [Some Description](https://github.com/robcaldecott/styleguidist-links)
 * 
 */

**Comment by sapegin** [comment]
This is obviously a bug: nobody will ever write links like that in comments and I believe it was working before. In any case it shouldn't produce errors.

**Comment by robcaldecott** [comment]
It was fine up to and including 6.0.33. Most MD engines seem to handle plain links.

**Comment by sapegin** [comment]
Must be connected with new version of markdown-to-jsx, but your snipped [seems to work fine](https://probablyup.github.io/markdown-to-jsx/). 

Feel free to send a pull request if you have time to investigate more and fix that.

**Comment by sapegin** [comment]
Nope, that's because Remark print links as auto links ;-(

**Comment by robcaldecott** [comment]
On 7.0.4 and still seeing this. :(  I'll try my sample.
